js/app.js

```diff
@@ -274,7 +274,7 @@ $(document).ready(function () {
     newElementEntry.find('.color-box').css('background-color', '#' + label.color);
 
     newElementEntry.find(':input[data-orig-val]').change(function () {
-      let $label = $(this).parents('.label-entry');
```

Comment:
Since you chose to use `$entry` for milestones, could you use `$entry` here, too, for labels?

Reply:
I thought you intended to differentiate between issues and milestones. On it.

Reply:
Yes. 

`$label` <-> `$milestones`

But, if `$entry` or `$card`, then let's parallel them

Reply:
> But, if $entry or $card, then let's parallel them

Precisely what I was thinking the first time around.